src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -115,6 +113,13 @@ public void run() {
             throwIfCancelled();
             updateMetadata(module, outputFile);
             throwIfCancelled();
+            for (Converter converter : converters) {
+                Path outputPath = converter.convert(outputFile);
```

Comment:
I'm not sure how this is different from what I'm doing here... I create a new Path pointing to the csv file that converter generates, and then I point outputFile (which is a path) to the new path